bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md

```diff
@@ -0,0 +1,66 @@
+#  Connecting Amazon OpenSearch Service to Microsoft Power BI Desktop
+
+>[!Note]
+
+>The following connector article is provided by Amazon, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the OpenSearch website and use the support channels there.
+
+## Summary
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
+| Products | Power BI Desktop |
+| Authentication Types Supported | Basic |
+
+## Prerequisites
+* Microsoft Power BI Desktop
+* [OpenSearch](https://docs-beta.opensearch.org/opensearch/install/index/)
+* [OpenSearch SQL ODBC driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/)

```

Comment:
minor: we are not using the beta links anymore
```suggestion
* [OpenSearch](https://opensearch.org/docs/latest/opensearch/install/index/)
* [OpenSearch SQL ODBC driver](https://opensearch.org/docs/latest/search-plugins/sql/odbc/)
```

Reply:
Thank you for the heads up. This has now been updated